core/foundation/src/TClassEdit.cxx

```diff
@@ -48,6 +48,11 @@ namespace {
             gInterpreterHelper->ShuttingDownSignal();
       }
    };
+
+   void RemoveSpace(std::string_view & s)
```

Comment:
```suggestion
   void RemoveLeadingSpace(std::string_view & s)
```

Reply:
I think that would be overdoing it slightly. If you look in the surroundings, there is already `RemoveStd`, and this function adds to the list of local helpers in the same spirit.
The reason it's in the .cxx in an anonymous namespace is to allow for inlining, but not affecting anything outside of this source file. It's local, and targeted to its job, but it shouldn't affect the rest of the world.